NawasNaziru/Banka
# #165344842 Merge form fields validation hotfix branch into master

What does this PR do?
 It adds validation to form fields

Description of Task to be completed?
  Have validation working in the following:
   - sign up form
   - sign in form
   - bank account creation form

How should this be manually tested?
 Open the form in browser and try submitting the form with empty values in required fields.

Any background context you want to provide?
 User data should be validated right at the front end without the need for round trip to server and back.

What are the relevant pivotal tracker stories?
 #165344842
